doc/BENCHMARKING.md

```diff
@@ -0,0 +1,45 @@
+## Benchmarking
+Currently, Mountpoint for Amazon S3 is an alpha release and we are focusing on delivering high throughput read support.  dIn order to make sure that new changes do not introduce performance regression we have created some performance benchmark  das a part of our CI using [fio](https://github.com/axboe/fio), an awesome opensource application for file system benchmarking.
```

Comment:
typo "dIn"
to be pedantic, when using describing something as open source, we hyphenate: "open-source application"

```suggestion

Currently, Mountpoint for Amazon S3 is an alpha release and we are focusing on delivering high throughput read support. In order to make sure that new changes do not introduce performance regression, we have created some performance benchmark as a part of our CI using [fio](https://github.com/axboe/fio), an awesome open-source application for file system benchmarking.
```